odoo/addons/test_main_flows/static/src/js/tour.js

```diff
@@ -407,14 +407,16 @@ tour.register('main_flow_tour', {
 }, {
 // Run the schedulers
     edition: "enterprise",
-    trigger: ".o_menu_sections a:contains('Schedulers')",
-    content: _t("Go to Schedulers"),
+    trigger: ".o_menu_sections a:contains('Operations')",
+    content: _t("Go to Scheduler"),
```

Comment:
This doesn't look right. The tour should now click on 'Schedulers' so it expands and then on 'Run Scheduler'.